yazi-config/preset/yazi.toml

```
@@ -12,6 +12,7 @@ linemode       = "none"
 show_hidden    = false
 show_symlink   = true
 scrolloff      = 5
+mouse          = true
```

```suggestion
mouse_events          = [ "click", "scroll" ]
```

This allows us to selectively ignore certain minor events, such as `touch`, `move`, `drag`, which can be manually enabled by the user when needed. Due to the high frequency of `move` events, this would be a significant optimization.